marker/jvm-marker/src/main/kotlin/spp/jetbrains/marker/jvm/formatter/ReflectiveValueFormatter.kt

```diff
@@ -0,0 +1,82 @@
+package spp.jetbrains.marker.jvm.formatter
+
+import com.intellij.ide.projectView.PresentationData
+import com.intellij.ui.SimpleTextAttributes
+import org.joor.Reflect
+import org.joor.ReflectException
+import spp.protocol.instrument.LiveVariable
+import java.time.*
+import java.util.*
+import kotlin.reflect.jvm.jvmName
+
+object ReflectiveValueFormatter {
+
+    private val supportedClasses: Set<String> = HashSet(
+        listOf(
+            Date::class.jvmName,
+            Duration::class.jvmName,
+            Instant::class.jvmName,
+            LocalDate::class.jvmName,
+            LocalTime::class.jvmName,
+            LocalDateTime::class.jvmName,
+            OffsetDateTime::class.jvmName,
+            OffsetTime::class.jvmName,
+            ZonedDateTime::class.jvmName
+        )
+    )
+
+    fun format(variable: LiveVariable, presentation: PresentationData) {
+        if (variable.liveClazz !in supportedClasses)
+            return
+        val obj = when (variable.value) {
+            is String -> variable.value
+            is List<*> -> setValues(variable.liveClazz, variable.value as List<Map<String, Any>>)
+            else -> null
+        }
+        obj.let { presentation.addText(" \"${it}\"", SimpleTextAttributes.REGULAR_ATTRIBUTES) }
+    }
+
+    private fun createInstance(liveClazz: String?): Reflect? {
```

Comment:
Move this to the `supportedClasses` and make it a map. I'd rather avoid using reflection to create the dummy objects. Something like:
```kt
val supportedTypes = mapOf(
    Date::class to Date(),
    Instant::class to Instant.now(),
    ...
)
```

Reply:
Changed as per suggestion. please check.